runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -149,15 +161,20 @@ public Dimension render(Graphics2D graphics)
 					Item item = (Item) current;
 					int itemId = item.getId();
 					int itemQuantity = item.getQuantity();
+					// Item does not support getting the name, so we need to get the ItemComposition instead
+					ItemComposition itemDefinition = client.getItemDefinition(itemId);
```

Comment:
The client has a cache for these but only for 64 items. Maybe we should cache this ourselves.